### ISSUE TITLE
ML: extraction non justifiée ?

### ISSUE THREAD

**Issue by tuduyen-nguyen** [raeell/SoftDTWForEcon#6]
Question : pourquoi dans `model/train_models_insee.py`, il y a une extraction des données (aussi bien train que val) ?
Lignes 34 à 37 : 
```
  X_val = x[-120:].to(device)
  y_val = y[-120:].to(device)
  X_tensor = x[:-120]
  Y_tensor = y[:-120]
```

**Comment by LaurineMir** [comment]
Mon idée (mais peut être à tord) c'est dans train le modèle de séparer les données d'entrainement en 2 (de manière arbitraire qu'on peut changer évidemment): une pour la train loss et une pour la validation loss. Et on utilise le test set pour faire les forecast

**Comment by tuduyen-nguyen** [comment]
Ah oui ok je vois. Pour plus de clarté je vais mettre ça dans une variable `train_samples` et à terme il faudra mettre ça dans `train_test_split`
